tests/test_cli.py

```diff
@@ -98,3 +98,79 @@ def test_end_to_end_multiple(capsys, tmp_path):
   assert not cap.out
   assert "foo.txt" in cap.err
   assert "Key[827bc3b2:EdPK] Signature verified" in cap.err
+
+
+def test_end_to_end_shortargs_armored(capsys, tmp_path):
+  from covert.__main__ import main
+  import sys
+  fname = tmp_path / "crypto.covert"
+
+  # Encrypt foo.txt into crypto.covert
+  sys.argv = "covert -eRao tests/keys/ssh_ed25519.pub".split() + [ str(fname) ] + [ 'tests/data/foo.txt' ]
+  ret = main()
+  cap = capsys.readouterr()
+  assert not ret
+  assert not cap.out
+  assert "foo" in cap.err
+
+  # Decrypt with key
+  sys.argv = "covert -di tests/keys/ssh_ed25519".split() + [ str(fname) ]
+  ret = main()
+  cap = capsys.readouterr()
+  assert not ret
+  assert not cap.out
+  assert "foo.txt" in cap.err
+
+
+def test_end_to_end_armormaxsize(capsys, tmp_path):
+  from covert.__main__ import main
+  import sys
+  fname = tmp_path / "test.dat"
+  outfname = tmp_path / "crypto.covert"
+
+  # Write 31 MiB on test.dat
+  with open(f"{fname}", "wb") as f:
+    f.seek(32505855)
+    f.write(b"\0")
+
+  # Encrypt test.dat with armor and no padding
+  sys.argv = f"covert -e --password verytestysecret --pad 0 -ao".split() + [ str(outfname), str(fname) ]
```

Comment:
Please use SSH keys in both of the new tests instead of a password because password hashing slows down the test run and keys are much faster. Also, the `--password` (undocumented CLI option) might get removed at some point because it is unsafe (password from command line is saved in shell command history and is also visible to all other users and programs on the system).